tensorflow_addons/metrics/cohens_kappa.py

```diff
@@ -89,8 +97,18 @@ def __init__(self,
         if weightage not in (None, 'linear', 'quadratic'):
             raise ValueError("Unknown kappa weighting type.")
 
+        if num_classes == 2:
+            self._update = self._update_binary_class_model
```

Comment:
Could you add a test case for `categorical_crossentropy` on a two class problem? I believe this will have issues.

Reply:
You want test cases for binary case, right?

Reply:
Done. Please check